pkg/pillar/scripts/device-steps.sh

```diff
@@ -19,7 +19,7 @@ LISPDIR=/opt/zededa/lisp
 LOGDIRA=$PERSISTDIR/IMGA/log
 LOGDIRB=$PERSISTDIR/IMGB/log
 AGENTS0="logmanager ledmanager nim nodeagent"
-AGENTS1="zedmanager zedrouter domainmgr downloader verifier identitymgr zedagent lisp-ztr baseosmgr wstunnelclient"
+AGENTS1="zedmanager zedrouter domainmgr downloader verifier identitymgr zedagent lisp-ztr baseosmgr wstunnelclient upgradeconverter"
```

Comment:
This doesn't work. The upgradeconverter needs to run before the first agent which needs the GlobalConfig, which is potentially ledmamager. I.e., before all the other agents.

Also, doesn't upgradeconverter need to be called with arguments and/or a json file to specify what to upgrade?